client/reader/following-manage/index.jsx

```diff
@@ -97,6 +103,7 @@ class FollowingManage extends Component {
 					<h1>{ translate( 'Manage Followed Sites' ) }</h1>
 				</MobileBackToSidebar>
 				{ searchResults.length === 0 && <QueryReaderFeedsSearch query={ sitesQuery } /> }
+				{ recommendedSites.length === 0 && <QueryReaderRecommendedSites seed={ this.state.seed } /> }
```

Comment:
how do we differentiate between "we asked for recs and none were found" and "i have not yet asked for recs and need to"?